src/mozanalysis/metrics.py

```diff
@@ -69,6 +73,7 @@ class DataSource:
     submission_date_column = attr.ib(default="submission_date", type=str)
     default_dataset = attr.ib(default=None, type=str | None)
     app_name = attr.ib(default=None, type=str | None)
+    group_id_column = attr.ib(default="profile_group_id", type=str)
```

Comment:
Should this and `client_id_column` above use `ExperimentalUnit.GROUP.value`? And elsewhere that we have the strings spelled out (like in `build_query`) we could use these values as well.

Reply:
Good callout.

Reply:
Fixed